fixtures/tests/_fixtures/test_warnings.py

```diff
@@ -18,9 +18,13 @@
 import fixtures
 
 
-class TestWarnings(testtools.TestCase, fixtures.TestWithFixtures):
+class TestWarningsCapture(testtools.TestCase, fixtures.TestWithFixtures):
 
     def test_capture_reuse(self):
+        # DeprecationWarnings are hidden by default in Python 3.2+, enable them
+        # https://docs.python.org/3/library/warnings.html#default-warning-filter
+        warnings.simplefilter("always")
```

Comment:
Does this leak configuration into other tests?  Maybe we should just have `tox` run the whole test suite with `PYTHONWARNINGS=always` or something like that?

Reply:
Yeah, I'm not sure. Given it's unit tests as opposed to something someone else will import, it didn't seem that big an issue. If we do this in tox, we're reliant on people running tox for unit tests to pass. Would it be weird to use our own fixtures (i.e. the new environment fixture) in the tests for other fixtures?

Reply:
I ended up using my own fixture.